packages/crypto/package.json

```diff
@@ -18,9 +18,10 @@
   "dependencies": {
     "@near-js/types": "workspace:*",
     "@near-js/utils": "workspace:*",
-    "borsh": "1.0.0",
     "@noble/curves": "1.2.0",
-    "randombytes": "2.1.0"
+    "borsh": "1.0.0",
+    "randombytes": "2.1.0",
+    "secp256k1": "5.0.0"
```

Comment:
not really, will fix it

Reply:
Thanks for the explanation!

Let's stick with `secp256k1` for now and handle that as a follow-up task. It will be much easier to deal with the issues you mention after we have working modules anyway.